inkscape/svg2shenzhen/prepare.py

```diff
@@ -62,19 +62,19 @@ def setDocumentSquare(self, width, height):
 
     def createLayer(self, layer_name):
         svg = self.document.xpath('//svg:svg',namespaces=inkex.NSS)[0]
-        layer = inkex.etree.SubElement(svg, 'g')
+        layer = etree.SubElement(svg, 'g')
         layer.set(inkex.addNS('label', 'inkscape'), '%s' % layer_name)
         layer.set(inkex.addNS('groupmode', 'inkscape'), 'layer')
         return layer
 
     def createWhitebg(self):
-        rect = inkex.etree.Element(inkex.addNS('rect','svg'))
+        rect = etree.Element(inkex.addNS('rect','svg'))
```

Comment:
This can now be `Rectangle.new(0, 0, w, h)` while style can be set with just `rect = {'fill': ...}`

Reply:
Same here